src/utils/predefined.js

```diff
@@ -1,34 +1,34 @@
 import { availableItems } from '@/utils/constants'
 
 // setting the base data
-let predefinedBars = [
+const predefinedBars = [
```

Comment:
If you make this a constant you need to move all of the additional declarations (adding the items below) inside this initial declaration. Which won't be a issue anymore, as the items it's just array of per-declared items. And when you move the `items` declaration the export predefinedBars right here.